en/config/actuators.md

```diff
@@ -1,65 +1,170 @@
 ```
 make px4_sitl gazebo_iris_ctrlalloc
+make px4_sitl gazebo_typhoon_h480_ctrlalloc
 ```
 :::
 
-After selecting an [airframe](../config/airframe.md) you will generally need to configure the specific geometry, assign actuators to outputs, and test the actuator response.
-This can be done in *QGroundControl*, under the **Vehicle Setup > Actuators** tab.
+Open the view in *QGroundControl* here: **"Q" (app menu) > Vehicle Setup > Actuators** (tab).
+
+## Overview
+
+The view has three sections:
+- [Geometry](#geometry): Set any configurable geometry-related parameters for the [selected airframe](../config/airframe.md).
+  This includes the number and relative position of motors, and the number, function, and properties of [control surfaces](#control-surfaces-geometry) (ailerons, rudders, etc.).
+- [Actuator Outputs](): Assign motors, control surfaces, and other actuators to specific flight controller outputs.
+- [Actuator Testing](): Test that motors and actuators are mapped to the correct outputs, and move as expected.
 
-The multicopter configuration screen looks like this.
+A quadcopter might have an setup screen similar to the one shown below.
+This defines a 4 motor copter with X-geometry, and maps each of the motors to DShot ESC.
+It also maps PWM400 AUX outputs for controlling a parachute and landing gear.
 
 ![Actuators MC (QGC)](../../assets/config/actuators/qgc_actuators_mc_aux.png)
 
-Note that some settings are hidden unless you select the **Advanced** checkbox in the top right corner.
+Note that only the most common settings are displayed by default.
+Select the **Advanced** checkbox in the top right corner to display all settings.
+
 
 ## Geometry
 
-The geometry section is used to configure any additional geometry-related settings for the selected [airframe](../config/airframe.md).
-The UI displays a customised view for the selected type; if there are no configurable options this may display a static image of the frame geometry, or nothing at all.
+The geometry section is used to set any configurable geometry-related parameters for the selected [airframe](../config/airframe.md).
+This includes the number and position of [motors](#motor-geometry), and the number, function, and properties of [control surfaces](#control-surfaces-geometry).
+For VTOL tiltrotor vehicles, it will also include the number and properties of [tilt servos](#motor-tilt-servo-geometry)
+
+The UI is customised for the selected airframe.
+Only motors and/or control surfaces that are relevant to the airframe type are displayed or available to assign.
+If there are no configurable geometry options for a particular frame, this section could conceivably be empty.
+
+:::note
+Not all aspects of a geometry are necessarily configurable through this interface.
+Factors that cannot be configured are hard coded for the current airframe. 
+:::
```

Comment:
What do you mean by this? Any examples?

Reply:
We know for a fact that you can't use a multicopter definition with a vtol, so there are some aspects of the geometry which are not captured. I assume some of them are irrelevant or can be tuned away. 

Or I could just be wrong. @bkueng ?